### ISSUE TITLE
Update the products listed on /firefox/ and /firefox/products/

### ISSUE THREAD

**Issue by craigcook** [mozilla/bedrock#9976]
## Description

The current Firefox landing page at /firefox/ features Lockwise, which is being phased out, and has no mention of Firefox Private Network, Firefox Relay, or Mozilla VPN. We should remedy that.

Mozilla VPN may be an awkward fit since it's not considered a "Firefox product" in terms of branding, but we already list Pocket which is also technically a separately branded product. We also include Mozilla VPN in the sticky promo on several pages as part of the product family, so there's precedent.

Firefox Private Network is still in beta so I'm not sure how much we want to promote it. It appears in the nav, but maybe we don't want to give it prime real estate on our product pages if it's not ready. If it IS ready for more marketing, we can start here.

There is also the separate Firefox Products page at /firefox/products/, which also lists Lockwise and not VPN, FPN, or Relay. We could potentially move that page up a level to just mozilla.org/products/ (which we used to have before we had a /firefox/ page and the URL now redirects to /firefox/, and the 'products' app in bedrock is now where the VPN landing page lives). Fully rearchitecting things right now may be out of scope, but we should at least just update the content to reflect the current family of products.

Several pages feature a sticky promo listing our family of products, and Lockwise is featured in that list. We should remove it if we're no longer promoting Lockwise.

tl; dr: We should remove Lockwise from all our "family of products" listings and add VPN, FPN, and Relay.

---

#### Success Criteria
- [ ] Remove Firefox Lockwise from product lists
- [ ] Add Mozilla VPN
- [ ] Add Firefox Private Network
- [ ] Add Firefox Relay

**Comment by craigcook** [comment]
See also https://github.com/mozilla/bedrock/issues/9905 ... this issue has a broader scope, it may supersede the previous one if we move ahead with doing more things. But if we limit to just the /firefox/ page the other issues covers that.

**Comment by alexgibson** [comment]
Closing as a duplicate of https://github.com/mozilla/bedrock/issues/10166
